docs/pages/management/register-services/register-services-join-token.mdx

```diff
@@ -0,0 +1,253 @@
+
+- Obtaining a join token
+- Running your Teleport service with the join token
+
+### Generate a token
+
+Teleport only allows access to services that have joined the cluster.
+
+Once a service joins, it receives a host certificate signed by the cluster's
+Auth Service. To receive a host certificate upon joining a cluster, a new
+Teleport host must present an **invite token**.
+
+An invite token also defines which role a new host can assume within a cluster:
+
+- `proxy`
+- `node`
+- `auth`
+- `app`
+- `db`
+- `kube`
+- `windowsdesktop`
+
+Administrators can generate tokens as they are needed. A token can be used
+multiple times until its time to live (TTL) expires.
+
+On your local machine, use the `tctl` tool to generate a new token. In the
+following example, a new token is created with a TTL of five minutes:
+
+```code
+# Generate a short-lived invite token for a new node:
+$ export INVITE_TOKEN=$(tctl nodes add --ttl=5m --roles=node | grep "invite token:" | grep -Eo "[0-9a-z]{32}")
+$ echo $INVITE_TOKEN
+# (=presets.tokens.first=)
+
+# You can also list all generated non-expired tokens:
+$ tctl tokens ls
+# Token                            Type            Expiry Time
+# ------------------------         -----------     ---------------
+# (=presets.tokens.first=)         service            25 Sep 18 00:21 UTC
+```
+
+<Details title="Provide your own token value">
+
+Rather than automatically generating a token, you can create one with a known
+value by using the `--token` flag:
+
+```code
+$ tctl nodes add --ttl=5m --roles=node,proxy --token=secret-value
+# The invite token: secret-value
+```
+
+The value of `--token` should be cryptographically secure. For example, you can
+create an SHA256 hash to reduce the risk that a malicious actor will produce
+a token with the same value as yours:
+
+```code
+$ head -n 1 /dev/random | sha256sum
+```
+
+</Details>
+
+<Details scope={["oss","enterprise"]} title="An insecure alternative: static tokens" scopeOnly={false} >
+
+<Notice type="Danger">
+Use short-lived tokens instead of long-lived static tokens.
+Static tokens are easier to steal, guess, and leak.
+</Notice>
+
+Static tokens are defined ahead of time by an administrator and stored in the
+auth server's config file:
+
+```yaml
+# Config section in `/etc/teleport.yaml` file for the auth server
+auth_service:
+    enabled: true
+    tokens:
+    # This static token allows new hosts to join the cluster as "proxy" or "node"
+    - "proxy,node:secret-token-value"
+    # A token can also be stored in a file. In this example the token for adding
+    # new auth servers are stored in /path/to/tokenfile
+    - "auth:/path/to/tokenfile"
+```
+</Details>
+
+### Start your service with the invite token
+
+Execute the following commands on the host where you will run your service so you
+can use the `INVITE_TOKEN` variable to start Teleport.
+
+```code
+$ export INVITE_TOKEN=<Var name="invite-token" />
```

Comment:
Lets use `JOIN_TOKEN` instead. An invite token is often used to refer to _user_ invites, and these tokens are not for users but for hosts.